flow360/component/results/case_results.py

```diff
@@ -422,21 +490,24 @@ class MaxResidualLocationResultCSVModel(ResultCSVModel):
     remote_file_name: str = pd.Field(CaseDownloadable.MAX_RESIDUAL_LOCATION.value, frozen=True)
 
 
+class ResultOperations:
+    @classmethod
+    def average_last_fraction(obj: ResultCSVModel, column, avarage_fraction):
+        df = obj.as_dataframe()
+        last_10_percent = df.tail(int(len(df) * avarage_fraction))
```

Comment:
```suggestion
        selected_fraction = df.tail(int(len(df) * avarage_fraction))
```

Reply:
addressed in: https://github.com/flexcompute/Flow360/pull/561/commits/7e8543c64dc6fbf027741165109bec18ccd51a6c